components/charts/SpendingTrendsChart.tsx

```
@@ -0,0 +1,223 @@
+                    0
+                  );
+
+                return {
+                  period: weekLabel,
+                  total,
+                  sortKey,
+                  weekIndex: index,
+                };
+              })
+              .sort((a, b) => a.sortKey - b.sortKey);
+          }
```

Biome (1.9.4)

[error] This else clause can be omitted because previous branches break early.

Unsafe fix: Omit the else clause.


(lint/style/noUselessElse)